solutions/fully-configurable/variables.tf

```diff
@@ -89,7 +88,7 @@ variable "key_protect_access_tags" {
 variable "rotation_interval_month" {
   type        = number
   description = "Specifies the key rotation time interval in months. Possible values: `1` through `12`. Applies only if an existing Key Protect or Hyper Protect Crypto Services instance is not specified."
-  default     = 12
+  default     = 3
```

Comment:
what is the impact here? Breaking change?

Reply:
It's just the change in the "default" value here, I don't think it might be a breaking change. We can discuss it further, if needed.